src/main/java/com/google/maps/model/OpeningHours.java

```diff
@@ -0,0 +1,58 @@
+package com.google.maps.model;
+
+public class OpeningHours {
+  /**
+   * openNow is a boolean value indicating if the place is open at the current time.
+   */
+  public Boolean openNow;
```

Comment:
Added java doc. Turning this from Boolean -> boolean leads to collapsing Closed and Unknown states into one.